### ISSUE TITLE
Unexpected results with day2night model

### ISSUE THREAD

**Issue by JaviMota** [junyanz/pytorch-CycleGAN-and-pix2pix#1210]
Hi

Im trying to run the day2night pretrained model but the results Im getting are not what I expected. 
I have followed the instructions for pytorch, tried to run the facades example an it works as expected.
I have replaced the facades model by the day2night model and the dataroot with the proper dataset. 
The direction is BtoA and the model pix2pix.
Am I doing something wrong?

Thanks.

**Comment by JunlinHan** [comment]
You may used the wrong direction if your results are roughly identical after translation. (Try A2B if that's the case)

**Comment by noobliang** [comment]
> Hi
> 
> Im trying to run the day2night pretrained model but the results Im getting are not what I expected.
> I have followed the instructions for pytorch, tried to run the facades example an it works as expected.
> I have replaced the facades model by the day2night model and the dataroot with the proper dataset.
> The direction is BtoA and the model pix2pix.
> Am I doing something wrong?
> 
> Thanks.

hello，how did you get the day2night pretrained model, and would you like share the model with me, please.  Limited by my GPU 1070ti, it dont support large size image，and training are very very slow,  thanks..

**Comment by JaviMota** [comment]
> You may used the wrong direction if your results are roughly identical after translation. (Try A2B if that's the case)

Hi, 
I have used both directions and the results are bad. The image translated is blurred and it doesn't reconstruct correctly the input image.

**Comment by JaviMota** [comment]
> > Hi
> > Im trying to run the day2night pretrained model but the results Im getting are not what I expected.
> > I have followed the instructions for pytorch, tried to run the facades example an it works as expected.
> > I have replaced the facades model by the day2night model and the dataroot with the proper dataset.
> > The direction is BtoA and the model pix2pix.
> > Am I doing something wrong?
> > Thanks.
> 
> hello，how did you get the day2night pretrained model, and would you like share the model with me, please. Limited by my GPU 1070ti, it dont support large size image，and training are very very slow, thanks..

Hi,

What I have done is to clone this repo and follow the instructions posted in the google colab pix2pix pytorch version.
https://colab.research.google.com/github/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/pix2pix.ipynb

I took the day2night pretrained model from scripts/download_pix2pix_model.sh
http://efrosgans.eecs.berkeley.edu/pix2pix/models-pytorch/

Finally I run the test instrcutions 
python3 test.py --dataroot ./datasets/day2night --direction BtoA --model pix2pix --name day2night_pix2pix
(I have tried --direction AtoB too)

Thanks for your reply.

**Comment by Daniil-Osokin** [comment]
Hi! I have also tried to get the similar results to the paper, and looks like it works :) For the **day2night** I have got for the val subset:
![Screenshot from 2020-12-22 14-34-39](https://user-images.githubusercontent.com/2491976/102884589-e8506a00-4462-11eb-91d9-0d7392e5455b.png)
The first scene is not very lucky, but quite frequent in the test set, so it may be the reason of unexpected results. But the rest two scenes look reasonable. For the **facades** results are similar to the paper:
![Screenshot from 2020-12-22 14-50-10](https://user-images.githubusercontent.com/2491976/102886092-88a78e00-4465-11eb-955e-8734122e6a7c.png)
And for the **sat2map** result with default input resolution (256x256) does not look good, but for the 512x512 (as outlined in the paper) result is similar:
![Screenshot from 2020-12-22 14-45-35](https://user-images.githubusercontent.com/2491976/102886458-331fb100-4466-11eb-84cf-2c436bd096da.png)
